src/main/java/com/vention/stock_market_share/repository/SecurityInfoRepository.java

```diff
@@ -130,6 +130,14 @@ public void delete(Long id) {
         }
     }
 
+    public boolean isValidInput(SecurityInfo securityInfo) {
+        return securityInfo.getUsername() != null
+                && securityInfo.getPassword() != null;
+
+
```

Comment:
for what this empty lines ? :)

Reply:
I just wanted to check if a user enters a null value while registering. Because I put a not null check on my database column

Reply:
136 137 lines for what?